packages/sdk-components-react/src/html-embed.ws.ts

```diff
@@ -17,6 +17,11 @@ export const meta: WsComponentMeta = {
 export const propsMeta: WsComponentPropsMeta = {
   props: {
     ...props,
+
+    executeScriptOnCanvas: {
```

Comment:
My guess is renaming the prop would be migration and not worth it?

Reply:
Yes